packages/form-js-editor/assets/form-js-editor.css

```diff
@@ -1,12 +1,22 @@
 .fjs-container .fjs-form-editor {
   display: flex;
+  flex: 1;
   flex-direction: row;
   height: 100%;
 }
 
 .fjs-container .fjs-form-container {
   flex: 1;
   height: 100%;
+  position: relative;
+}
+
+.fjs-container .fjs-form {
+  position: absolute;
```

Comment:
What is this good for? Anything particular this fixes?

Reply:
I believe this needs adjustment for the logo:

![capture ZaCcgZ_optimized](https://user-images.githubusercontent.com/58601/112835871-254b7e80-909a-11eb-9906-71d866a8ced5.gif)

Reply:
Both the form and the properties panel are supposed to scroll vertically. Taken from: https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/PropertiesContainer.less#L43

Reply:
I'll have a look and fix this.

Reply:
https://github.com/bpmn-io/form-js/pull/40/commits/c694236b5e2ea715ce40f09e2a733a4ac0a8dd5b

https://github.com/bpmn-io/form-js/pull/40/commits/ed25a5481c14cdec019d402d77243459860ddbb2